catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -125,6 +129,18 @@
   private static final Supplier<Boolean> ZERO_PAGESIZE_COMPATIBILTY =
       () -> Boolean.valueOf(System.getProperty(ZERO_PAGESIZE_COMPATIBILITY_PROPERTY));
 
+  @VisibleForTesting
+  static final String SOLR_COMMIT_NRT_METACARDTYPES = "solr.commit.nrt.metacardTypes";
+
+  private final Set<String> commitNrtMetacardType =
+      Sets.newHashSet(accessProperty(SOLR_COMMIT_NRT_METACARDTYPES, "").split(","));
+
+  @VisibleForTesting
+  static final String SOLR_COMMIT_NRT_COMMITWITHINMS = "solr.commit.nrt.commitWithinMs";
+
+  private final int commitNRtCommitWithinMs =
```

Comment:
It would not hurt to add a minimum but I do not think we should limit the max.  It is possible to override the Solr soft and hard commit times to be even higher.